CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 This changelog only lists major version changes (breaking), or minor changes important enough to list them here. Check individual releases (tags) and their commits to see unlisted changes.
 
 
+### v7.0.0 (2019-11-05)
+
+#### Restrict Node.js engine versions
+Node.js versions `>= 12.2.0 <= 13.0.0` are now rejected, since they'll throw an error (`ERR_REQUIRE_ESM`) due to the usage of `@grrr/gulpfile`.
```

Comment:
You mean `@grrr/utils` in this sentence.

Reply:
I do! 👍